### ISSUE TITLE
sumneko_lua fails to install

### ISSUE THREAD

**Issue by cdimitroulas** [williamboman/nvim-lsp-installer#696]
### Problem description

When installing the sumneko_lua server, the installation fails.

### Neovim version (>= 0.6)

0.7.0

### Operating system/version

Ubuntu 18.04

### I've recently downloaded the latest plugin version of both nvim-lsp-installer and nvim-lspconfig

- [ ] Yes

### Affected language servers

sumneko_lua

### Actual behavior

The server fails to install

### Expected behavior

The server should install successfully

### LspInstallInfo output

```shell
Pending servers (1) 
    ◍ sumneko_lua (failed)
        ...ler/lua/nvim-lsp-installer/core/managers/github/init.lua:28: Failed to fetch latest release from GitHub API.
```

### Installation log

```shell
[INFO  Fri 13 May 2022 17:28:41 BST] ...ed/nvim-lsp-installer/lua/nvim-lsp-installer/ui/init.lua:732: Starting install server_name="sumneko_lua", requested_version=""
[ERROR Fri 13 May 2022 17:28:41 BST] ...im-lsp-installer/lua/nvim-lsp-installer/core/process.lua:153: Failed to spawn process. cmd="gh", err="ENOENT: no such file or directory"
[INFO  Fri 13 May 2022 17:28:42 BST] ...r/lua/nvim-lsp-installer/core/managers/github/client.lua:76: Failed to find latest release. repo="sumneko/vscode-lua", opts={
  include_prelease = false
}
[ERROR Fri 13 May 2022 17:28:42 BST] ...installer/lua/nvim-lsp-installer/core/installer/init.lua:79: Installation failed, name="sumneko_lua", error="...ler/lua/nvim-lsp-installer/core/managers/github/init.lua:28: Failed to fetch latest release from GitHub API."
[INFO  Fri 13 May 2022 17:28:42 BST] ...ed/nvim-lsp-installer/lua/nvim-lsp-installer/ui/init.lua:751: Installation completed server_name="sumneko_lua", success=false
```

### Healthcheck

```shell
nvim-lsp-installer: require("nvim-lsp-installer.health").check()
========================================================================
## nvim-lsp-installer report
  - OK: neovim version >= 0.7.0
  - WARNING: **Go**: not available
  - WARNING: **cargo**: not available
  - WARNING: **Ruby**: not available
  - WARNING: **RubyGem**: not available
  - WARNING: **Composer**: not available
  - WARNING: **PHP**: not available
  - WARNING: **julia**: not available
  - OK: **sh**: `Ok`
  - OK: **bash**: `GNU bash, version 4.4.20(1)-release (x86_64-pc-linux-gnu)`
  - OK: **tar**: `tar (GNU tar) 1.29`
  - OK: **gzip**: `gzip 1.6`
  - OK: **wget**: `GNU Wget 1.19.4 built on linux-gnu.`
  - OK: **python3**: `Python 3.6.9`
  - OK: **node**: `v16.15.0`
  - OK: **curl**: `curl 7.58.0 (x86_64-pc-linux-gnu) libcurl/7.58.0 OpenSSL/1.1.1 zlib/1.2.11 libidn2/2.0.4 libpsl/0.19.1 (+libidn2/2.0.4) nghttp2/1.30.0 librtmp/2.3`
  - OK: **java**: `Ok`
  - OK: **javac**: `javac 11.0.7`
  - OK: **npm**: `8.5.5`
  - OK: **pip3**: `pip 19.1.1 from /home/cdimitroulas/.local/lib/python3.6/site-packages/pip (python 3.6)`
```

### Screenshots

![Screenshot from 2022-05-13 17-21-34](https://user-images.githubusercontent.com/20640274/168328394-b46a3a4d-371d-4d57-9d1d-bc11ef24a4e8.png)

**Comment by cdimitroulas** [comment]
Seems like it's trying to use a `gh` command which I don't have installed

**Comment by cdimitroulas** [comment]
This is also happening to me for the `omnisharp` server

**Comment by williamboman** [comment]
Hello! You're probably being rate limited by GitHub's API. I just merged https://github.com/williamboman/nvim-lsp-installer/pull/697 which helps diagnose this. Could you try pulling the latest version and run `:checkhealth nvim-lsp-installer` again 🙏 ?

**Comment by hassty** [comment]
Happens to me as well. The problem seems to be in this statement
https://github.com/williamboman/nvim-lsp-installer/blob/245f1bece5b906bd7f2a11e684c3619055b6ce1d/lua/nvim-lsp-installer/core/managers/github/client.lua#L66

**Comment by williamboman** [comment]
> Happens to me as well. The problem seems to be in this statement

Ah! That was changed earlier today - I reverted it just now so should work now. I did not think through the logic well enough 😅

**Comment by williamboman** [comment]
Fixed in #699.

**Comment by cdimitroulas** [comment]
Thanks for the quick turnaround!
